rustfmt.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
A version of this fix has already been applied. Can you please check out the HEAD version of the file and either break this line out into another PR or remove it from the diff?